qemu/tests/qemu_img_convert_before_snapshot.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+from qemu.tests.qemu_disk_img import generate_base_snapshot_pair
+
+
+def run(test, params, env):
+    """
+    qemu-img create a snapshot on a running base image.
```

Comment:
'Create snapshot based on the target qcow2 image converted from raw source image'